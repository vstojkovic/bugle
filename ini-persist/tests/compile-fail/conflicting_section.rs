use ini_persist::IniLoad;

#[derive(IniLoad)]
#[ini(section = "This")]
#[ini(section = "That")]
struct Foo {
    argle: u8,
    bargle: String,
}

#[derive(IniLoad)]
#[ini(general)]
#[ini(section = "Specific")]
struct Bar {
    glop: u8,
    glyf: String,
}

#[derive(IniLoad)]
#[ini(section = "Specific")]
#[ini(general)]
struct Baz {
    olle: u8,
    bolle: String,
}

fn main() {
}
