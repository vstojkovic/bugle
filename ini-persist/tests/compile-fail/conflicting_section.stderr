error: conflicting section specified
 --> tests/compile-fail/conflicting_section.rs:5:7
  |
5 | #[ini(section = "That")]
  |       ^^^^^^^^^^^^^^^^

error: conflicting section specified
  --> tests/compile-fail/conflicting_section.rs:13:7
   |
13 | #[ini(section = "Specific")]
   |       ^^^^^^^^^^^^^^^^^^^^

error: conflicting section specified
  --> tests/compile-fail/conflicting_section.rs:21:7
   |
21 | #[ini(general)]
   |       ^^^^^^^
